contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -35,25 +40,46 @@ def register_options(cls, register):
       register('--version', advanced=True, default='6.9.1',
                help='Node distribution version.  Used as part of the path to lookup the '
                     'distribution with --binary-util-baseurls and --pants-bootstrapdir')
+      register('--package-manager', advanced=True, default='npm', fingerprint=True,
+               choices=NodeDistribution.VALID_PACKAGE_MANAGER_LIST.keys(),
+               help='Default package manager config for repo. Should be one of [npm,yarnpkg]')
```

Comment:
The option is marked `advanced`, so it doesn't show in the CLI help unless you pass `./pants help-all --advanced`, or use `./pants options` (the latter can be easier to grok).